x-pack/plugins/enterprise_search/public/applications/analytics/components/add_analytics_collections/add_analytics_collection_logic.ts

```diff
@@ -51,16 +62,37 @@ export const AddAnalyticsCollectionLogic = kea<
 >({
   actions: {
     createAnalyticsCollection: () => {},
-    setInputError: (inputError: string | boolean) => ({ inputError }),
+    setInputError: (inputError) => ({ inputError }),
```

Comment:
nitpick: missing type on `inputError` argument

Reply:
It takes from generic kea type. So IDE can see it. Do I need to add it?